package.json

```diff
@@ -64,8 +64,10 @@
     "karma-junit-reporter": "0.4.1",
     "karma-ng-html2js-preprocessor": "0.2.1",
     "minimatch": "3.0.0",
+    "mkdirp": "^0.5.1",
     "mocha": "2.4.5",
     "mock-fs": "3.8.0",
+    "promise": "^7.1.1",
```

Comment:
I disagree. I get "ReferenceError: promise is not defined" if I don't declare it in gulpfile.js.